MaxText/kernels/ragged_attention.py

```diff
@@ -0,0 +1,302 @@
+"""Kernels for ragged attention."""
+
+import functools
+
+import jax
+from jax import lax
+from jax.experimental import pallas as pl
+from jax.experimental.pallas import tpu as pltpu
+import jax.numpy as jnp
+import numpy as np
+import common_types
+
+from jax.experimental import shard_map
+
+
+DEFAULT_MASK_VALUE = -0.7 * float(np.finfo(np.dtype("float32")).max)
```

Comment:
Include comment here why large negative value is the appropriate filler as opposed to zero (value corresponds to logit, will be softmaxed)

Reply:
This is already used in attentions.py and mentioned elsewhere in the codebase. So I was just reusing what was already established. But we can add some info to it either here or move it to some common place so it's not redefined.

Reply:
Latest revision moves `DEFAULT_MASK_VALUE` to be defined in `common_types.py` and I added a small comment about why it's used.